src/components/Home.js

```diff
@@ -1,10 +1,32 @@
 /* eslint no-console: 0 */
 import React from 'react'
 
+import getStateName from '../util/usa'
+import LocationSelect from './LocationSelect'
+import { slugify } from '../util/text'
+import { updateFilters, updateFiltersAndUrl } from '../actions/filterActions'
+
 import MapSvg from '../../img/usa-map.svg'
 
-const Home = () => {
-  const clickMap = e => console.log(e.target.getAttribute('id'))
+const scrollToBottom = () => window.scrollTo(0, window.outerHeight * 1.5)
+
+const Home = ({ appState, dispatch, location }) => {
+  const { crime, place } = appState.filters
+  const isButtonDisabled = !!(crime && place) || false
+
+  const handleMapClick = e => {
+    const id = e.target.getAttribute('id')
```

Comment:
we should add a `if (!id) return` line (because u can click in the oceans around the state which we want to ignore